cli/kata-env.go

```diff
@@ -82,6 +82,7 @@ type HypervisorInfo struct {
 	Path              string
 	BlockDeviceDriver string
 	Msize9p           uint32
+	MemorySlots       uint32
```

Comment:
Thanks! Can you also bump `formatVersion` in this file to make it clear to users that the format changed slightly?

Reply:
@jodh-intel - did the bump happen? If so, CI's are all green...

Reply:
Not yet - let me add a label...